iseq.c

```diff
@@ -3814,6 +3814,25 @@ rb_vm_insn_decode(const VALUE encoded)
     return insn;
 }
 
+// Returns a boolean indicating whether or not the iseq accesses self.
+bool
+rb_vm_iseq_reads_self(const rb_iseq_t *iseq)
```

Comment:
Doesn't this name look like a predicate very much?

Reply:
Yes, it does. Should I add a `_p`, or?